tests/framework/vm_config_network.json

```diff
@@ -0,0 +1,42 @@
+{
+    "boot-source": {
+        "kernel_image_path": "vmlinux.bin",
+        "boot_args": "console=ttyS0 reboot=k panic=1 pci=off",
+        "initrd_path": null
+    },
+    "drives": [
+        {
+        "drive_id": "rootfs",
+        "path_on_host": "bionic.rootfs.ext4",
+        "is_root_device": true,
+        "partuuid": null,
+        "is_read_only": false,
+        "cache_type": "Unsafe",
+        "io_engine": "Sync",
+        "rate_limiter": null
+        }
+    ],
+    "machine-config": {
+        "vcpu_count": 2,
+        "mem_size_mib": 1024,
+        "smt": false,
+        "track_dirty_pages": false
+    },
+    "cpu-config": null,
+    "balloon": null,
+    "network-interfaces": [
+        {
+            "iface_id": "1",
```

Comment:
if we use eth0 here we could later consider this file as a good reference for getting started guide to start Firecracker with network.

Reply:
This makes sense, I think this would be in a separate PR.